build_tools/aws-sdk-code-generator/lib/aws-sdk-code-generator.rb

```diff
@@ -76,6 +76,25 @@
 require_relative 'aws-sdk-code-generator/code_builder'
 require_relative 'aws-sdk-code-generator/gem_builder'
 
+# RBS
+if RUBY_VERSION >= '3.2'
```

Comment:
Should this be > 3.0? Is this because of dependencies? Please add a more detailed comment about why this needs to be true. We should try to open this up if possible.

Reply:
Should we just fail/raise an error if we don't have a ruby version that supports rbs generation?

Our release tools / code generator does not need to support the same ruby versions as the general SDK gems - and I'd be worried about the possibility of running code generation on an older ruby and silently NOT generating signatures - and once we release them, I think removing them would be considered a breaking change.